bentoml/deployment/store.py

```diff
@@ -146,17 +156,40 @@ def delete(self, name, namespace):
                     "Deployment '%s' in namespace: '%s' is not found" % name, namespace
                 )
 
-    def list(self, namespace, filter_str=None, labels=None, offset=None, limit=None):
+    def list(
+        self,
+        namespace,
+        operator=None,
+        labels=None,
+        offset=None,
+        limit=None,
```

Comment:
might be good to add a default limit as well, like `200`, and default offset `0`

Reply:
that way you can remove the `if limit` check below as well